<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtWidgetsApplication1Class</class>
 <widget class="QMainWindow" name="QtWidgetsApplication1Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>300</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Test App</string>
  </property>
  <widget class="PaintedWidget" name="centralWidget">
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionTriangle"/>
   <addaction name="actionRect"/>
   <addaction name="actionCircle"/>
   <addaction name="separator"/>
   <addaction name="actionLink"/>
   <addaction name="actionMove"/>
   <addaction name="actionTrash"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="actionLoad"/>
  </widget>
   <actiongroup name="RadioButtons">
   <action name="actionTriangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="QtWidgetsApplication1.qrc">
     <normaloff>:/QtWidgetsApplication1/Images/Triangle.png</normaloff>:/QtWidgetsApplication1/Images/Triangle.png</iconset>
   </property>
   <property name="text">
    <string>Triangle</string>
   </property>
   <property name="toolTip">
    <string>Draws a triangle</string>
   </property>
  </action>
  <action name="actionRect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="QtWidgetsApplication1.qrc">
     <normaloff>:/QtWidgetsApplication1/Images/Box.png</normaloff>:/QtWidgetsApplication1/Images/Box.png</iconset>
   </property>
   <property name="text">
    <string>Rect</string>
   </property>
   <property name="toolTip">
    <string>Draws a rectangle</string>
   </property>
  </action>
  <action name="actionCircle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="QtWidgetsApplication1.qrc">
     <normaloff>:/QtWidgetsApplication1/Images/Circle.png</normaloff>:/QtWidgetsApplication1/Images/Circle.png</iconset>
   </property>
   <property name="text">
    <string>Circle</string>
   </property>
   <property name="toolTip">
    <string>Draws a circle</string>
   </property>
  </action>
  <action name="actionLink">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="QtWidgetsApplication1.qrc">
     <normaloff>:/QtWidgetsApplication1/Images/Link.png</normaloff>:/QtWidgetsApplication1/Images/Link.png</iconset>
   </property>
   <property name="text">
    <string>Link</string>
   </property>
   <property name="toolTip">
    <string>Links two shapes</string>
   </property>
  </action>
  <action name="actionMove">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="QtWidgetsApplication1.qrc">
     <normaloff>:/QtWidgetsApplication1/Images/Pan.png</normaloff>:/QtWidgetsApplication1/Images/Pan.png</iconset>
   </property>
   <property name="text">
    <string>Move</string>
   </property>
   <property name="toolTip">
    <string>Moves a shape</string>
   </property>
  </action>
  <action name="actionTrash">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="QtWidgetsApplication1.qrc">
     <normaloff>:/QtWidgetsApplication1/Images/Trash.png</normaloff>:/QtWidgetsApplication1/Images/Trash.png</iconset>
   </property>
   <property name="text">
    <string>Trash</string>
   </property>
   <property name="toolTip">
    <string>Removes a shape</string>
   </property>
  </action>
   </actiongroup>
   <action name="actionSave">
   <property name="icon">
    <iconset resource="QtWidgetsApplication1.qrc">
     <normaloff>:/QtWidgetsApplication1/Images/Save.png</normaloff>:/QtWidgetsApplication1/Images/Save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Saves shapes to file</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="icon">
    <iconset resource="QtWidgetsApplication1.qrc">
     <normaloff>:/QtWidgetsApplication1/Images/Load.png</normaloff>:/QtWidgetsApplication1/Images/Load.png</iconset>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
   <property name="toolTip">
    <string>Loads shapes from file</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PaintedWidget</class>
   <extends>QWidget</extends>
   <header>PaintedWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="QtWidgetsApplication1.qrc"/>
 </resources>
 <connections/>
</ui>
